desktop/apps/editorial_features/components/venice_2017/templates/body.jade

```diff
@@ -1,8 +1,10 @@
 .venice-body--article
   .venice-body__left
+    h2 About the Film
     +item_description(section)
     +headset_prompt
   .venice-body__right
+    h2 About the Film
```

Comment:
There are two of them because the order of things is different in the mobile view -- I'm hiding one of them based on screen size